cmd/agent/app/options/options.go

```diff
@@ -102,6 +111,9 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.StringVar(&o.KarmadaContext, "karmada-context", "", "Name of the cluster context in karmada control plane kubeconfig file.")
 	fs.StringVar(&o.ClusterName, "cluster-name", o.ClusterName, "Name of member cluster that the agent serves for.")
 	fs.StringVar(&o.ClusterNamespace, "cluster-namespace", DefaultKarmadaClusterNamespace, "Namespace in the control plane where member cluster secrets are stored.")
+	fs.StringVar(&o.ClusterProvider, "cluster-provider", "", "Provider of the joining cluster.")
+	fs.StringVar(&o.ClusterRegion, "cluster-region", "", "The region of the joining cluster.")
+	fs.StringVar(&o.ClusterZone, "cluster-zone", "", "The zone of the joining cluster")
```

Comment:
Please don't combine these flags with this PR. This PR focus on reporting the secret thing.

You can do it by a separated PR. By the way, the `--cluster-zone` might need to update to `--cluster-zones` and it accepts a string. (The API part also needs an update).

Reply:
ok

Reply:
We can add `provider`, `region`, `zone` flags and related code in the next pr.

Reply:
Hi @CharlesQQ Why are the three flags still in this PR?